### DISCUSSION TITLE
DOSXYZnrc

### DISCUSSION THREAD

**Discussion by mona399** [nrc-cnrc/EGSnrc#1187] [General]
![image](https://github.com/user-attachments/assets/31f18887-efae-4a36-8335-96ecca53e063)
if build LINAC and set the target in z=0cm, SSD=100cm, and scoring plane after the component ends in z=100cm and after running BEAMnrc I will use it in DOSXYZnrc how can determine the distance to the source in isocenter? can help me, please.

**Comment by rtownson** [comment]
If you score the phase-space in BEAMnrc at z=100cm, then place the `distance from source to isocenter` as 0 for isocenter `0,0,0`, then your particles will indeed start at `0,0,0`. This distance is just the distance from the isocenter to the scoring plane.

**Comment by mona399** [comment]
Thanks for reply, If you build a 30cm*30cm*30cm phantom using DOSXZYnrc and
minimum boundary in x axis and y axis =-15cm while on z axis minimum
boundary is 0 what is the appropriate isocenter in this case?

**Reply by rtownson** [reply]
The isocenter should be realistic - where ever it actually is for the experiment that you're modelling. Then position your phase-space using the distance to the source appropriately.
